core/workspace_svg.js

```diff
@@ -1482,7 +1482,9 @@ Blockly.WorkspaceSvg.prototype.updateToolbox = function(tree) {
     this.options.languageTree = tree;
     this.flyout_.show(tree.childNodes);
   }
-  this.toolbox_.position();
```

Comment:
I see. Happy to move the call to right after line 1477 to avoid the extra if.

As for the call to positioning the flyout... I'm hesitant to add something when there is no clear purpose. Is there any behavior you are concerned about might be bugged without that call? Logically, as I understand it, loading a new set of blocks into the toolbox should not actually reposition it. There is no option to switch between toolbox layouts here, so its not like it will shift anchors (aka top, left, etc). So it seems the toolbox position would not change. One edge-case might be the scroll-bar, as loading a new set of blocks might adjust how it looks given the different number of blocks... but testing this it seems to work just fine: I can see the scrollbar adjust according to the blocks in the toolbox after loading. So, again, I'm hesitant to blindly add a call when I don't see a reason nor can I test if anything has changed, but if you want me to I'm happy to do it (but would prefer not to). Please let me know, I'll make the other change for now.